### PR TITLE
Updating no songs yet page to match the mocks

### PR DESCRIPTION
**Pull request by escobarjonatan**
image

### CODE REVIEW

**Review by dmkirshon — CHANGES_REQUESTED** [review]
Just one comment on button color

Also not required on this change but just trying to understand the table styling, do we just need to create some `theme` based properties for the table to keep the rows and headers consistent?

File: src/pages/home/library/NoSongsYet.tsx [hunk 1]
```diff
@@ -1,39 +1,38 @@
...
-        />
-      </IconButton>
-    </Box>
+      <Button
+        onClick={ () => navigate("/home/upload-song") }
+        variant="secondary"
+        width="compact"
+      >
+        Upload your first song
+      </Button>
```
[the remark this answers is not in the document]
Comment (escobarjonatan) [reply src/pages/home/library/NoSongsYet.tsx]: Good catch, I didnt think that was an option bc i saw it used without it in other places lol the irony is that i created the button component...

**Comment by escobarjonatan** [comment]
>Also not required on this change but just trying to understand the table styling, do we just need to create some theme based properties for the table to keep the rows and headers consistent?

Not sure if a theme like would help, but maybe we could create a Table component that has sub components and we could reuse. I thought about doing that but I figured we can do that when we start to get other table requests to see what we need to make configurable 🤷🏽‍♂️

**Review by dmkirshon — APPROVED** [review]
With the updates, LGTM

**Pull request merged**